assets/search/components/TopicEditor.tsx

```diff
@@ -110,6 +110,8 @@ class TopicEditor extends React.Component<IProps, IState> {
     }
 
     componentDidUpdate(prevProps: Readonly<IProps>) {
+        this.reloadFolders(this.props.topic != null && this.props.topic.is_global);
```

Comment:
you could also do ` this.reloadFolders(this.props.topic?.is_global === true)`

current version is fine too